### PR TITLE
Move the selection between the native and Python implementations of lzo and lz4 to dissect.util

### PR DESCRIPTION
**Pull request by pyrco**

### CODE REVIEW

**Comment by pyrco** [comment]
This PR depends on https://github.com/fox-it/dissect.util/pull/51

File: README.md [hunk 1]
```diff
@@ -17,7 +17,17 @@ Information on the supported Python versions can be found in the Getting Started
...
-This module is also automatically installed if you install the `dissect` package.
+This project decompresses lzo compressed file systems and can use the faster, native (C-based) lzo implementation when
+installed, instead of the slower pure Python implementation provided by `dissect.util`. To use these faster
+implementations, install the package with the lzo extra:
+
+```bash
+pip install "dissect.btrfs[lzo]"
+```
+
+Unfortunately there is no binary python-lzo wheel for PyPy installations on Windows, so it won't be installed there.
```
Comment (Schamper) [inline README.md:28]:
```suggestion
Unfortunately there is no binary `python-lzo` wheel for PyPy installations on Windows, so it won't be installed there.
```

**Review by Schamper — CHANGES_REQUESTED** [review]

File: README.md [hunk 2]
```diff
@@ -17,7 +17,17 @@ Information on the supported Python versions can be found in the Getting Started
...
-This module is also automatically installed if you install the `dissect` package.
+This project decompresses lzo compressed file systems and can use the faster, native (C-based) lzo implementation when
+installed, instead of the slower pure Python implementation provided by `dissect.util`. To use these faster
+implementations, install the package with the lzo extra:
+
+```bash
+pip install "dissect.btrfs[lzo]"
+```
+
+Unfortunately there is no binary `python-lzo wheel` for PyPy installations on Windows, so it won't be installed there.
```
Comment (Schamper) [inline README.md:28]:
```suggestion
Unfortunately there is no binary `python-lzo` wheel for PyPy installations on Windows, so it won't be installed there.
```

File: dissect/btrfs/stream.py [hunk 3]
```diff
@@ -6,13 +6,9 @@
 from typing import TYPE_CHECKING, BinaryIO, NamedTuple
 from uuid import UUID
 
+from dissect.util import lzo
```
Comment (Schamper) [inline dissect/btrfs/stream.py:9]:
```suggestion
from dissect.util.compression import lzo
```
I think I like this a bit better.

**Review by Schamper — CHANGES_REQUESTED** [review]

**Review by Schamper — DISMISSED** [review]

**Review by Schamper — APPROVED** [review]

**Pull request merged**